api-server/rockcraft.yaml

```diff
@@ -0,0 +1,117 @@
+name: kfp-api
+summary: An image for using Kubeflow pipelines API
+description: An image for using Kubeflow pipelines API
+version: "2.0.0-alpha.7_20.04_1"
+base: ubuntu:20.04
+license: Apache-2.0
+platforms:
+  amd64:
+
+services:
+  apiserver:
+    summary: "pipelines api-server service"
+    override: merge
+    command: /bin/apiserver --config=/config --sampleconfig=/config/sample_config.json -logtostderr=true
+    startup: enabled
+    user: ubuntu
+
+parts:
+  security-team-requirement:
+    plugin: nil
+    override-build: |
+      mkdir -p ${CRAFT_PART_INSTALL}/usr/share/rocks
+      (echo "# os-release" && cat /etc/os-release && echo "# dpkg-query" && dpkg-query -f '${db:Status-Abbrev},${binary:Package},${Version},${source:Package},${Source:Version}\n' -W) > ${CRAFT_PART_INSTALL}/usr/share/rocks/dpkg.query
+
+  builder:
+    plugin: go
+    source: https://github.com/kubeflow/pipelines.git
+    source-tag: 2.0.0-alpha.7
+    build-snaps:
+      - go/1.17/stable
+    build-environment:
+      - GO111MODULE: "on"
+    override-build: |-
+      go build -o apiserver backend/src/apiserver/*.go
+      cp apiserver $CRAFT_STAGE
+      cp -r backend/src/apiserver/config/ $CRAFT_STAGE/config
+      ./hack/install-go-licenses.sh
+      $GOBIN/go-licenses check ./backend/src/apiserver
+      $GOBIN/go-licenses csv ./backend/src/apiserver > $CRAFT_STAGE/licenses.csv && \
+      diff $CRAFT_STAGE/licenses.csv backend/third_party_licenses/apiserver.csv && \
+      $GOBIN/go-licenses save ./backend/src/apiserver --save_path $CRAFT_STAGE/NOTICES
+    build-packages:
+      - cmake
+      - clang
+      - musl-dev
+      - openssl
```

Comment:
Nit but I'd shift this above the `override-build` alongside other deps and setup